shape.lic

```diff
@@ -140,13 +144,26 @@ class Shape
     shape(command)
   end
 
-  def finish
-    waitrt?
-    stow_item(right_hand)
-    fput("get my #{@material} lumber")
-    fput("put my #{@material} lumber in my #{@bag}")
+   def finish
+    case @finish
+    when /log/
+      logbook_item
+    when /stow/
+      stow_item(@noun)
+    when /trash/
+      dispose_trash(@noun)
+    end
     exit
   end
+  
+  def logbook_item
```

Comment:
This looks nearly identical to the method in `sew.lic`. Is there any benefit to moving this to a `common-crafting` method?